src/field_spec.h

```diff
@@ -65,7 +65,8 @@ Each field has a unique id that's a power of two, so we can filter fields
 by a bit mask.
 Each field has a type, allowing us to add non text fields in the future */
 typedef struct FieldSpec {
-  char* name;
+  char *name;
+  char *path;
```

Comment:
Maybe better to use `as_name` instead of `path` (as `alias` is taken).

Reply:
`path` is the field in hash or path in JSON so `as_name` would be misleading.